selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/organization/OrganizationMembersTest.java

```diff
@@ -32,93 +31,82 @@
 import org.testng.annotations.Test;
 
 /** @author Sergey Skorik */
-public class OrganizationTest {
+public class OrganizationMembersTest {
+  private static final String PRE_CREATED_ORG_NAME = generate("organization", 5);
+  private static final String NEW_ORG_NAME = generate("organization", 5);
 
-  private static final String PRE_CREATED_ORG_NAME = generate("orgX", 6);
-  private static final String NEW_ORG_NAME = generate("orgY", 6);
-
-  private List<String> emailsList;
   private OrganizationDto organization;
+  private String memberEmail;
+
+  @Inject
+  @Named("admin")
+  private TestOrganizationServiceClient testOrganizationServiceClient;
 
   @Inject private OrganizationListPage organizationListPage;
   @Inject private OrganizationPage organizationPage;
   @Inject private NavigationBar navigationBar;
   @Inject private AddOrganization addOrganization;
   @Inject private AddMember addMember;
   @Inject private Loader loader;
-
-  @Inject
-  @Named("admin")
-  private TestOrganizationServiceClient testOrganizationServiceClient;
-
   @Inject private Dashboard dashboard;
   @Inject private TestUser testUser;
   @Inject private AdminTestUser adminTestUser;
 
   @BeforeClass
   public void setUp() throws Exception {
-    emailsList = Arrays.asList(testUser.getEmail());
+    memberEmail = testUser.getEmail();
 
     organization = testOrganizationServiceClient.create(PRE_CREATED_ORG_NAME);
     testOrganizationServiceClient.addAdmin(organization.getId(), adminTestUser.getId());
-
     dashboard.open(adminTestUser.getName(), adminTestUser.getPassword());
   }
 
   @AfterClass
   public void tearDown() throws Exception {
-    testOrganizationServiceClient.deleteByName(PRE_CREATED_ORG_NAME);
-    testOrganizationServiceClient.deleteByName(NEW_ORG_NAME);
+    for (OrganizationDto organization : testOrganizationServiceClient.getAll())
+      testOrganizationServiceClient.deleteById(organization.getId());
```

Comment:
Such code can affect organizations which are created in parallel threads of test execution.